docs/getting-started/installation-and-upgrade/resources/upgrade-cert-manager.md

```diff
@@ -97,7 +99,7 @@ In order to upgrade cert-manager, follow these instructions:
     helm install \
       cert-manager jetstack/cert-manager \
       --namespace cert-manager \
-      --version v1.11.0
+      --version v1.13.1
```

Comment:
```suggestion
```

Couldn't include it in the suggestion, but the `\` in `--namespace cert-manager \` needs to be dropped too.